src/backend/pipelines/lcm_lora.py

```diff
@@ -23,11 +23,14 @@ def get_lcm_lora_pipeline(
     lcm_lora_id: str,
     use_local_model: bool,
     torch_data_type: torch.dtype,
+    pipeline_args={},
 ):
-    pipeline = DiffusionPipeline.from_pretrained(
+    # pipeline = DiffusionPipeline.from_pretrained(
```

Comment:
Why do we need to switch AutoPipelineForText2Image?

Reply:
Just like with LCM, I use _AutoPipelineForText2Image_ to simplify ControNet pipeline creation. _AutoPipelineForText2Image_ will create a _StableDiffusionControlNetPipeline_ if ControlNet is enabled and it will create a _StableDiffusionPipeline_ when not enabled.